core-navigation/build.gradle.kts

```diff
@@ -28,6 +28,7 @@ dependencies {
     api(libs.androidx.hilt.navigation.compose)
     api(libs.androidx.navigation.compose)
 
+    implementation(libs.androidx.tracing.ktx)
```

Comment:
Files in `core-navigation` don't seem to use any imports from `tracing`, can it be removed?